### ISSUE TITLE
[BUG] Price Formatting doesn't work correctly

### ISSUE THREAD

**Issue by SladeHazard** [snowgears/shopbugs#397]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Type of bug

Other unexpected behaviour

### Environment

```markdown
- Server Version: 1.12.2 Paper
- Shop Version: 1.8.4.2-legacy
```

### Server startup log

N/A

### Shop config files

N/A

### Bug description

Formatting of balance on shop signs does not work properly for the billion.

If I put 30B on the Buy section price. it converts it to 30$ instead of $30,000,000,000 but it only works correctly when I put 30G. people are not familiar with the character "G" can you please fix it so it works with the "B" = billion as well. This is my formatting config:

This formats suffixes for large numbers like (5,000 = 5k) or (5,000,000 = 5M) #
minimumValue = minimum value to start formatting strings #
priceSuffixes:
  minimumValue: 10000
  k:
    enabled: true
    value: 1000
  M:
    enabled: true
    value: 1000000
  B:
    enabled: true
    value: 1000000000
  T:
    enabled: true
    value: 1000000000000
  P:
    enabled: true
    value: 1000000000000000
  E:
    enabled: true
    value: 1000000000000000000

### Steps to reproduce

create a buy or sell shop  and put B character after a number in the price section for example 30B 

[shop]
3
30B
buy

### Expected behaviour

the price should be 30 billion

### Actual behaviour

the price is $30

**Comment by iLemon** [comment]
Can confirm, having the same issue.

**Comment by Firosee** [comment]
This happens for me when trying T (trillions), it sets it to be $21.47B, and same for any formatting above it (Q, E, etc)

**Comment by Firosee** [comment]
^

**Comment by OstlerDev** [comment]
> This happens for me when trying T (trillions), it sets it to be $21.47B, and same for any formatting above it (Q, E, etc)

Unfortunately there is a limitation of our internal variables that cap the max price at 21.47 Billion, so using amounts like Trillions and higher will not work.

**Comment by SladeHazard** [comment]
> > This happens for me when trying T (trillions), it sets it to be $21.47B, and same for any formatting above it (Q, E, etc)
> 
> Unfortunately there is a limitation of our internal variables that cap the max price at 21.47 Billion, so using amounts like Trillions and higher will not work.

Yeah you just summarized the bug, nice I guess..?

**Comment by OstlerDev** [comment]
Bug is fixed in the upcoming version v1.9.1.0

**Comment by OstlerDev** [comment]
Released 1.9.1.0
